scripts/compare-deposits/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/nspcc-dev/neo-go/scripts/compare-deposits
+
+go 1.22.3
```

Comment:
Decrease go version to match the rest of our repos?

Reply:
Fixed!